2021-09-13T14:55:41.4312000+08:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-09-13T14:55:41.4430198+08:00  [INF] Hosting environment: "Development" (c3307c92)
2021-09-13T14:55:41.4449631+08:00  [INF] Content root path: "C:\Users\Jan\source\repos\PS- clone\PaymentSystemApi\PaymentSystem" (b5d60022)
2021-09-13T14:55:55.5321120+08:00 80000012-0009-ff00-b63f-84710c7967bb [ERR] PS.Domain.Entities.InvalidAccountException: Error Getting balance
   at PS.Application.Services.TransactionService.GetAcctBalance() in C:\Users\Jan\source\repos\PS- clone\PaymentSystemApi\PS.Application\Services\TransactionService.cs:line 37
   at PaymentSystem.Controllers.Account.GetAcctBalance() in C:\Users\Jan\source\repos\PS- clone\PaymentSystemApi\PaymentSystem\Controllers\Account.cs:line 23
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at PaymentSystem.Middleware.ErrorHandlerMiddleware.Invoke(HttpContext context, ILogger`1 logger) in C:\Users\Jan\source\repos\PS- clone\PaymentSystemApi\PaymentSystem\Middleware\ErrorHandlerMiddleware.cs:line 22 (db6d9cc3)
2021-09-13T15:15:46.8657464+08:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-09-13T15:15:46.8778613+08:00  [INF] Hosting environment: "Development" (c3307c92)
2021-09-13T15:15:46.8798901+08:00  [INF] Content root path: "C:\Users\Jan\source\repos\PS- clone\PaymentSystemApi\PaymentSystem" (b5d60022)
2021-09-13T15:16:38.2348931+08:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-09-13T15:16:38.2566374+08:00  [INF] Hosting environment: "Development" (c3307c92)
2021-09-13T15:16:38.2582749+08:00  [INF] Content root path: "C:\Users\Jan\source\repos\PS- clone\PaymentSystemApi\PaymentSystem" (b5d60022)
2021-09-13T15:16:43.9163499+08:00 80000017-0001-ff00-b63f-84710c7967bb [ERR] PS.Domain.Entities.InvalidAccountException: Error Getting balance
   at PS.Application.Services.TransactionService.GetAcctBalance() in C:\Users\Jan\source\repos\PS- clone\PaymentSystemApi\PS.Application\Services\TransactionService.cs:line 39
   at PaymentSystem.Controllers.Account.GetAcctBalance() in C:\Users\Jan\source\repos\PS- clone\PaymentSystemApi\PaymentSystem\Controllers\Account.cs:line 23
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at PaymentSystem.Middleware.ErrorHandlerMiddleware.Invoke(HttpContext context, ILogger`1 logger) in C:\Users\Jan\source\repos\PS- clone\PaymentSystemApi\PaymentSystem\Middleware\ErrorHandlerMiddleware.cs:line 22 (9e520c73)
2021-09-13T15:22:22.7798434+08:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-09-13T15:22:22.8000845+08:00  [INF] Hosting environment: "Development" (c3307c92)
2021-09-13T15:22:22.8016504+08:00  [INF] Content root path: "C:\Users\Jan\source\repos\PS- clone\PaymentSystemApi\PaymentSystem" (b5d60022)
2021-09-13T15:25:48.5062106+08:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-09-13T15:25:48.5126736+08:00  [INF] Hosting environment: "Development" (c3307c92)
2021-09-13T15:25:48.5144825+08:00  [INF] Content root path: "C:\Users\Jan\source\repos\PS- clone\PaymentSystemApi\PaymentSystem" (b5d60022)
2021-09-13T16:57:00.9170567+08:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-09-13T16:57:00.9349964+08:00  [INF] Hosting environment: "Development" (c3307c92)
2021-09-13T16:57:00.9627224+08:00  [INF] Content root path: "C:\Users\Jan\source\repos\PS- clone\PaymentSystemApi\PaymentSystem" (b5d60022)
2021-09-13T16:57:49.2952387+08:00 8000001d-0001-ff00-b63f-84710c7967bb [ERR] System.Text.Json.JsonException: A possible object cycle was detected which is not supported. This can either be due to a cycle or if the object depth is larger than the maximum allowed depth of 32.
   at System.Text.Json.ThrowHelper.ThrowInvalidOperationException_SerializerCycleDetected(Int32 maxDepth)
   at System.Text.Json.JsonSerializer.Write(Utf8JsonWriter writer, Int32 originalWriterDepth, Int32 flushThreshold, JsonSerializerOptions options, WriteStack& state)
   at System.Text.Json.JsonSerializer.WriteAsyncCore(Stream utf8Json, Object value, Type inputType, JsonSerializerOptions options, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Mvc.Formatters.SystemTextJsonOutputFormatter.WriteResponseBodyAsync(OutputFormatterWriteContext context, Encoding selectedEncoding)
   at Microsoft.AspNetCore.Mvc.Formatters.SystemTextJsonOutputFormatter.WriteResponseBodyAsync(OutputFormatterWriteContext context, Encoding selectedEncoding)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeResultAsync>g__Logged|21_0(ResourceInvoker invoker, IActionResult result)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResultFilterAsync>g__Awaited|29_0[TFilter,TFilterAsync](ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResultExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.ResultNext[TFilter,TFilterAsync](State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeResultFilters()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at PaymentSystem.Middleware.ErrorHandlerMiddleware.Invoke(HttpContext context, ILogger`1 logger) in C:\Users\Jan\source\repos\PS- clone\PaymentSystemApi\PaymentSystem\Middleware\ErrorHandlerMiddleware.cs:line 22 (35a6dab1)
